styles/paper_network.json

```diff
@@ -0,0 +1,33 @@
+[
```

Comment:
The format in this JSON file is not uniform. Some lines (for example, lines 5, 16, 17) have a space character after `:`, some don't (such as lines 6, 8, 9), etc. You may want to correct them (because JSON format is infamously inflexible sometimes).